src/CTA.WebForms2Blazor/ControlConverters/ControlConverter.cs

```diff
@@ -9,7 +9,38 @@ namespace CTA.WebForms2Blazor.ControlConverters
 {
     public abstract class ControlConverter
     {
+        protected class Attribute
+        {
+            public string Name { get; }
+            public string Value { get; }
+
+            public Attribute(string name, string value)
+            {
+                Name = name;
+                Value = value;
+            }
+
+            public override string ToString()
+            {
+                return Name + "=" + Value;
+            }
+
+            public bool Equals(Attribute other)
+            {
+                if (other is null)
+                {
+                    return false;
+                }
+                return this.Name == other.Name;
+            }
+            public override bool Equals(object obj) => Equals(obj as Attribute);
+            public override int GetHashCode() => (Name).GetHashCode();
```

Comment:
Hashcode.Combine(Name) would be better here for good spread and also for matching CTA in general. Please apply this throughout rest of project also.